docker-entry.sh

```diff
@@ -56,31 +58,37 @@ if [ $VERSIONINFO -eq 1 ]; then
   echo "using ${MIGRIDVERSION}"
 fi
 
+# Run self-test if requested to ease support
+if [ $CHECKCONF -eq 1 ]; then
```

Comment:
I think that unless it is hiding somewhere else, that you are missing variable declaration CHECKCONF=0 before this conditional check if the c) option is not specified. I would think it would make sense to declare it at the same place as the KEEPALIVE=0 declaration

Reply:
Thanks! I actually hit and fixed that issue inline while testing on `bench`, but I can see I must have forgotten to roll the addition of `CHECKCONF=0` and the added `-c` flag to the `docker-entry.sh` call for `migrid-volume-init` in docker-compose through here :nerd_face:

Reply:
Shellcheck does not flag those particular unquoted values - probably because quoting makes little difference/sense when using the numerical comparison (`-eq`).
In particular the uninitialized `CHECKCONF` would still lead to a hard error with such quoting:
```
if [ "$CHECKCONF" -eq 1 ]; then echo 'Yes!'; fi
bash: [: : integer expression expected
```